src/elements/MetricsSelector/MetricsSelector.js

```diff
@@ -258,8 +288,16 @@ const MetricsSelector = ({ maxSelectionNumber, metrics, name, onSelect, preselec
                   </FieldArray>
                 </ul>
                 <FormOnChange name={name} handler={handleOnChange} />
-                <div data-testid="metrics-selector-counter" className="metrics-selector-counter">
-                  {`${formState.values.metrics?.length ?? 0}/${maxSelectionNumber}`}
+                <div className="metrics-selector__footer">
+                  <div data-testid="metrics-selector-counter" className="metrics-selector-counter">
+                    {`${formState.values.metrics?.length ?? 0}/${maxSelectionNumber}`}
+                  </div>
+                  <Button
+                    id="metrics-selector-apply-btn"
```

Comment:
you don't use this `id` anywhere, you can delete it

Reply:
done.